### DISCUSSION TITLE
Julia Code is Weirdly Rendered for Pluto.jl Notebooks

### DISCUSSION THREAD

**Discussion by jeremiahpslewis** [github-linguist/linguist#5637] [Classification] [answered]
See here: https://github.com/JuliaPluto/PlutoTest.jl/blob/main/src/notebook.jl

Initial code is nicely rendered and then rest is not syntax highlighted

![IMG_0690](https://user-images.githubusercontent.com/4462211/140608915-95892e34-0f95-4aac-9ce6-950605f31ef2.jpeg)

![IMG_0692](https://user-images.githubusercontent.com/4462211/140608889-8f334563-9f92-4ffd-89b6-34d0d3c57f6d.jpeg)

**Comment by jeremiahpslewis** [comment]
This seems to have been fixed here: https://github.com/JuliaEditorSupport/atom-language-julia/issues/244 and is not an issue in VSCode (even in Github VSCode file browser thing)

**Reply by lildude** [reply, marked as answer]
This is still going to be a problem with the grammar as we can see if we test with the latest version of it, completely independently of Linguist, [here](https://github-lightshow.herokuapp.com/?utf8=✓&scope=from-url&grammar_format=auto&grammar_url=https%3A%2F%2Fgithub.com%2FJuliaEditorSupport%2Fatom-language-julia%2Fblob%2Fmaster%2Fgrammars%2Fjulia.cson&grammar_text=&code_source=from-url&code_url=https%3A%2F%2Fgithub.com%2FJuliaPluto%2FPlutoTest.jl%2Fblob%2Fmain%2Fsrc%2Fnotebook.jl&code=%3C%3Fphp%0D%0A%0D%0A%23%5BAttribute%5D%0D%0Aclass+Test+%7B%0D%0A%09function+__construct%28array+%24data%29+%7B%7D%0D%0A%7D%0D%0A%0D%0Afunction+method%28%23%5BTest%28%5B%27value%27+%3D%3E+true%5D%29%5D+%24data%29+%7B%7D+%2F%2F+%3C-----+here).

This isn't likely to be an issue in VSCode because it appears to use a completely different grammar; it uses https://github.com/julia-vscode/julia-vscode 

You'll need to take this up with the upstream grammar maintainers.

**Reply by jeremiahpslewis** [reply]
Ok, thanks!
